aws_marketplace/using_model_packages/preprocessing-audio-data-using-a-machine-learning-model/preprocessing-audio-data-using-a-machine-learning-model.ipynb

```diff
@@ -0,0 +1,894 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    transformer = model.transformer(instance_count=32, 
</pre><p>32 is a lot of instances, but you're only using 2xl. Could you use fewer larger instances? If the existing setup is preferred, fine to leave as-is.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3061/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>For some reason changing the instance type to a more powerful one does not improve the performance by a big margin. However, increasing the number of instances does. With 12 instances, it takes around an hour and with 32 the time taken is around 30 minutes. Hence, I'll probably leave it as it is. I have dropped comments to indicate that this parameter can be changed to tune runtime and cost.</p>